firmware/library/L2_Utilities/util.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <cstdint>
+
+namespace Utility
+{
+static uint32_t Random(uint32_t min, uint32_t max);
+static void SetRandomSeed();
+static uint32_t BitExtract(uint32_t extractee, uint8_t start_bit,
```

Comment:
Would prefer the word target, `extractee` doesn't seem as readable. That goes for the other implementations as well.